src/python/turicreate/test/test_image_similarity.py

```diff
@@ -287,6 +291,60 @@ def test_save_and_load(self):
             print("Export coreml passed")
 
 
+class ImageSimilarityTestWithKwargs(unittest.TestCase):
+    @classmethod
+    def setUpClass(self, input_image_shape=(3, 224, 224), model="resnet-50"):
+        """
+        The setup class method for the basic test case with all default values.
+        """
+        self.feature = "awesome_image"
```

Comment:
I don't think you need the majority of the code in this `setUpClass`. It looks like the only member variable that's actually getting used is `self.model`. I think everything else should be removed.